.gitlab-ci.yml

```diff
@@ -56,9 +63,23 @@ Debian9:
 
   <<: *testing_script
 
+Debian10:
+  variables:
+    distro: debian:10
+    playbook: test.yml
+
+  <<: *testing_script
+
 Ubuntu1604_NetworkTopology:
   variables:
     distro: 'ubuntu:16.04'
     playbook: test_networktopology.yml
 
   <<: *testing_script
+
+Ubuntu1804_NetworkTopology:
+  variables:
+    distro: 'ubuntu:18.04'
+    playbook: test_networktopology.yml
```

Comment:
one little request: could you simply replace `Ubuntu1604_NetworkTopology` with this block instead of adding a new one? The network topology module can be tested just once. We don't want the build time to increase too much.

Reply:
Sure! On it! :smile:

Reply:
Done! Added build for `fedora:27` as well.
Note: `fedora:28` not added because it will requires more changes,
I think i should create another issue for it! :smile:

Reply:
@atb00ker yes please create a new issue for `fedora28`